x-pack/plugins/searchprofiler/public/application/components/profile_query_editor/profile_query_editor.tsx

```diff
@@ -141,21 +118,6 @@ export const ProfileQueryEditor = memo(() => {
                 />
               </EuiFormRow>
             </EuiFlexItem>
-            <EuiFlexItem grow={false}>
```

Comment:
dont we wanna leave the EuiIconButton here instead? Otherwise we will endup with a different version of the UI for 8.17.. given that both main and [the 8.16 backport](https://github.com/elastic/kibana/pull/208480/files) have the IconButton instead

Reply:
Yes, I'll create a new PR that's about the same as the 8.16 backport.